sdk/core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure_core"
-version = "0.8.0"
+version = "0.10.0"
```

Comment:
Shouldn't this be 0.9.0?

Reply:
I commented on the PR on this.  I intentionally bumped azure_core to 0.10.0 to reduce the burden of having to keep track of different version numbers for different crates.